.pipelines/pipeline.user.windows.yml

```diff
@@ -120,12 +120,7 @@ build:
             - 'modules\launcher\Plugins\Microsoft.Plugin.WindowWalker\Wox.Infrastructure.dll'
             - 'modules\launcher\Plugins\Microsoft.Plugin.WindowWalker\Wox.Plugin.dll'
             - 'modules\launcher\Plugins\Microsoft.Plugin.WindowWalker\Telemetry.dll'
-            - 'modules\launcher\Plugins\Microsoft.Plugin.Service\Microsoft.Plugin.Service.dll'
-            - 'modules\launcher\Plugins\Microsoft.Plugin.Service\Wox.Plugin.dll'
-            - 'modules\launcher\Plugins\Microsoft.Plugin.Service\PowerToysInterop.dll'
-            - 'modules\launcher\Plugins\Microsoft.Plugin.Service\Microsoft.PowerToys.Settings.UI.Lib.dll'
-            - 'modules\launcher\Plugins\Microsoft.Plugin.Service\ManagedTelemetry.dll'
-            - 'modules\launcher\Plugins\Microsoft.Plugin.Service\ManagedCommon.dll'
+            - 'modules\launcher\Plugins\Microsoft.PowerToys.Run.Plugin\Microsoft.PowerToys.Run.Plugin.Service.dll'
```

Comment:
I think that using the namespace for this folder is pointless, all the plugins in `modules\launcher\Plugins\` are `Microsoft.PowerToys.Run` plugins, we can just call it with the plugin name, in this case `Service`

Reply:
We will update the folders names for the other plugins in a different PR.

Reply:
This is the pipeline that is working on output dir.
Should I remove the namespace from both output dir and install dir?

From
`\x64\Release\modules\launcher\Plugins\Microsoft.PowerToys.Run.Plugin.Service`
`C:\Program Files\PowerToys\modules\launcher\Plugins\Microsoft.PowerToys.Run.Plugin.Service`
to
`\x64\Release\modules\launcher\Plugins\Service`
`C:\Program Files\PowerToys\modules\launcher\Plugins\Service`

Reply:
Yes that would be great and it would be used as a reference for fixing all the other plugins.

Reply:
@enricogior
Do we have a standard for the naming and ist this written down in the dev docs?